Makefile

```
@@ -0,0 +1,20 @@
+GO ?= go
+GOFMT ?= gofmt
+
+.PHONY: check fmt-check test vet
+
+check: fmt-check test vet
+
+fmt-check:
```

checkmake (0.3.2)

[warning] Target body for "fmt-check" exceeds allowed length of 5 lines (6).

(maxbodylength)